app/helpers/application_helper.rb

```diff
@@ -113,6 +113,14 @@ def fa_icon(icon, attributes = {})
     content_tag(:i, nil, attributes.merge(class: class_names.join(' ')))
   end
 
+  def material_symbol(icon, attributes = {})
+    inline_svg_tag(
+      "#{icon}.svg",
```

Comment:
I think it makes sense to explicitly reference `material-icons/400-24px/` here (and possibly allow for different weight or size), if only to avoid potential ambiguity with other assets.

Reply:
Adjusted to add the size here.